pages/oku/_location.vue

```diff
@@ -10,26 +10,58 @@
 <script>
 import OKU from '~components/oku-circ'
 import moment from 'moment'
+import { mapState } from 'vuex'
 
 export default {
   data () {
     return {
-      okuLocation: this.$route.params.location,
-      currentTime: moment().format('MMM D / h:mm A')
+      okuLocation: this.$route.params.location
     }
   },
   components: {
     'oku-circ': OKU
   },
+  computed: {
+    ...mapState({
+      currentTime: state => moment(state.time.now).format('MMM D / h[<span class="blink">:</span>]mm A')
+    })
+  },
   async fetch ({ store, params }) {
     await store.dispatch('laptops/fetchStatus', 'olin')
     await store.dispatch('phoneChargers/fetchStatus', 'olin')
     await store.dispatch('laptops/fetchStatus', 'uris')
     await store.dispatch('phoneChargers/fetchStatus', 'uris')
   },
-  layout: 'oku'
+  layout: 'oku',
+  mounted () {
+    // Sync current time every 10 seconds
+    // -- ideally, this would happen within the store itself (as part of the action),
+    // -- but encountered a bug when attempting this via Nuxt. No issue in vanilla Vue.
+    setInterval(() => {
+      this.$store.dispatch('time/sync')
+    }, 1000 * 10)
+
+    // Update equipment status every minute
+    setInterval(() => {
+      this.$store.dispatch('laptops/fetchStatus', 'olin')
+      this.$store.dispatch('phoneChargers/fetchStatus', 'olin')
+      this.$store.dispatch('laptops/fetchStatus', 'uris')
+      this.$store.dispatch('phoneChargers/fetchStatus', 'uris')
+    }, 1000 * 60)
+  }
 }
 </script>
 
-<style>
+<style lang="scss">
```

Comment:
Works for me. Just checked and [can't see any good reason to use prefixes](http://caniuse.com/#feat=css-animation) for this animation, so I simplified it even further.

I don't feel strongly about the blinking colon, so if you'd prefer to drop it, just let me know.